src/Tabs.js

```diff
@@ -81,13 +83,12 @@ const defaultProps = {
 };
 
 function getDefaultActiveKey(children) {
-  let defaultActiveKey;
+  let defaultActiveKey = null;
   forEach(children, child => {
-    if (defaultActiveKey == null) {
+    if (defaultActiveKey === null) {
```

Comment:
this change is _not_ correct. if the `eventKey` on a child is `undefined` (because it's not defined), then we end up calculating a `defaultActiveKey` here of `undefined`, because this check will now fail.

please undo the changes to this function. it's good to have documentation for the `defaultActiveKey` prop, but this change is wrong

Reply:
not sure how it can be correct to return an undefined key but agree this would be a breaking change